client.go

```diff
@@ -180,7 +182,27 @@ func runStreamedCommand(img, cmd string, getError bool) error {
 // It returns the ID of the container spawned from the image.
 // Note well: the container is NOT deleted when the given command terminates.
 func runCommandInContainer(img string, cmd []string, dst io.Writer) (string, error) {
-	return startContainer(img, cmd, true, dst)
+	id, err := createContainer(img, cmd)
+	if err != nil {
+		log.Println(err)
+		return "", err
+	}
+
+run_cmd:
```

Comment:
Use a regular `for` loop with a boolean value or something else. Please refrain from using a `goto` statement unless it's *really* necessary.